src/components/App.jsx

```diff
@@ -17,12 +17,18 @@ import { SPLIT_STATUS } from '../ducks/splits';
 import GALogAdapter from '../lib/ga-log-adapter';
 import GoogleLogger from '../lib/GoogleLogger';
 import GeordiLogAdapter from '../lib/geordi-log-adapter';
+import { checkLoginUser } from '../ducks/login';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.googleLogger = null;
+
+    if (!props.initialised) {
+      props.dispatch(checkLoginUser());
+    }
```

Comment:
I moved this to App without realizing later that I don't think it's necessary. I'm open to move it back if we decide to do so.

Reply:
Although I still need to test this, moving `checkLoginUser()` to the root `App` component _makes sense_ if we want user state to be a root-level concern. (We do.)

IIRC, the reason that `checkLoginUser()` originally sat in `Header` was because it was part of zoo-reduxify's `AuthHeader` component, which handled the user state logistics in a single, easy to grok component.